City-of-Helsinki/drupal-infofinland
# UHF-9527 D10 and PHP8.2

D10 update

## How to install

Run this: `make down && docker-compose down -v && git checkout dev && make up && composer i && make drush-sync-db drush-deploy && git checkout UHF-9527 && make up && composer i && make drush-deploy`

## How to test
After the install process is done, D10 should be updated. Log in and test it out.
